apps/NES/Makefile.coq.local

```diff
@@ -0,0 +1,3 @@
+install-extra::
```

Comment:
It might be nicer to move all of the predicates into `NES.Db`, rather than install `nes.elpi` because
* clients who want to use the predicates need two `Elpi Accumulate` commands rather than one, plus an `Extra Dependency`
* (minor) BedRock's version of `coqdep` does not seem to understand `From elpi.apps Extra Dependency "nes.elpi" as nes.`